core/Convert.php

```diff
@@ -445,4 +445,42 @@ public static function base64url_decode($val) {
 			true
 		);
 	}
+
+	/**
+	 * Convert a php.ini value (eg: 512M) to bytes
+	 *
+	 * @todo unit tests
+	 *
+	 * @param string $iniValue
+	 * @return int
+	 */
+	public static function ini2bytes($iniValue) {
```

Comment:
Feels a bit icky having these in Convert. I'm not sure where they should go. We don't want Convert to be just a grab-bag of presenters though - ideally it would be "safely convert between types", not "parse and/or present a type".